Makefile

```diff
@@ -11,12 +12,24 @@ commands :
 
 ## pdf        : re-generate PDF
 pdf :
-	${LATEX} ${STEM}
-	${BIBTEX} ${STEM}
-	${LATEX} ${STEM}
-	${LATEX} ${STEM}
-	${LATEX} ${CHK}
-	${LATEX} ${CHK}
+	${LATEX} ${ROBU}
+	${BIBTEX} ${ROBU}
+	${LATEX} ${ROBU}
+	${LATEX} -output-directory=dist ${ROBU}
+	${LATEX} -jobname=S1_Checklist ${CHK}
+	${LATEX} -jobname=S1_Checklist -output-directory=dist ${CHK}
```

Comment:
It changes the name of the output file to S1_checklist, according to PLOS submission guidelines, and puts the final documents into the dist folder.